Demo/Setup.cs

```diff
@@ -58,12 +58,12 @@ public static bool Init()
                 try
                 {                    
                     ConnectionUtil.Session = new ApexSharp().
-                         SalesForceUrl("https://login.salesforce.com/")
+                         SalesForceUrl("https://test.salesforce.com/")
                         .AndSalesForceApiVersion(40)
                         
-                        .WithUserId("SF Login ID")
-                        .AndPassword("Password")
-                        .AndToken("Token")
+                        .WithUserId("allison@healthclx.com.test")
+                        .AndPassword("healthclxer007")
```

Comment:
I don't think that was intentional?
You hardly want to put your password in the source code repository :)

Reply:
definitely not.  thanks